### PR TITLE
Rebase to 1.9.1

### PR DESCRIPTION
**Pull request by deads2k**
/assign mfojtik
/assign soltysh

@soltysh the audit options probably deserve better wiring.

I wonder if it is this simple....

### CODE REVIEW

File: vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go [hunk 1]
```diff
@@ -421,7 +421,7 @@ func GetAvailableResources(clientBuilder controller.ControllerClientBuilder) (ma
 // CreateControllerContext creates a context struct containing references to resources needed by the
 // controllers such as the cloud provider and clientBuilder. rootClientBuilder is only used for
 // the shared-informers client and token controller.
-func CreateControllerContext(s *options.CMServer, rootClientBuilder, clientBuilder controller.ControllerClientBuilder, stop <-chan struct{}) (ControllerContext, error) {
+func createControllerContext(s *options.CMServer, rootClientBuilder, clientBuilder controller.ControllerClientBuilder, stop <-chan struct{}) (ControllerContext, error) {
```
Comment (soltysh) [inline vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/controllermanager.go]: I assume this carry replaces https://github.com/openshift/origin/commit/ce08d2fdb2990ff8b1b7fcad79236f87c0a621a6 ?

File: pkg/cmd/server/kubernetes/master/master_config.go [hunk 2]
```diff
@@ -819,7 +820,7 @@ func GetAuditConfig(auditConfig configapi.AuditConfig) (audit.Backend, auditpoli
 
 		// webhook configuration, only when config file was provided
 		if len(auditConfig.WebHookKubeConfig) > 0 {
-			webhook, err := auditwebhook.NewBackend(auditConfig.WebHookKubeConfig, string(auditConfig.WebHookMode), auditv1beta1.SchemeGroupVersion)
+			webhook, err := auditwebhook.NewBackend(auditConfig.WebHookKubeConfig, string(auditConfig.WebHookMode), auditv1beta1.SchemeGroupVersion, pluginwebhook.NewDefaultBatchBackendConfig())
```
Comment (soltysh) [inline pkg/cmd/server/kubernetes/master/master_config.go]: It's ok to start with. But I'll open a followup and we'll externalize these configs.

**Comment by deads2k** [comment]
Looks like one simple unit test and this is ready

**Comment by mfojtik** [comment]
LGTM

good job @deads2k !

Will wait for @soltysh to tag this as this carries fixes to auditing.

**Comment by kargakis** [comment]
@openshift/sig-master

**Comment by kargakis** [comment]
@openshift/sig-master

**Comment by deads2k** [comment]
according to prow, the extended builds are failing on rebase pulls too.

Downward API is a known flake: https://github.com/openshift/origin/issues/17882

**Comment by liggitt** [comment]
> according to prow, the extended builds are failing on rebase pulls too.
>
> Downward API is a known flake: #17882

agree. merging to reopen the queue.

**Pull request merged**